src/providers/hooks/useObservations.js

```diff
@@ -17,10 +17,15 @@ const useObservations = ( ): Object => {
   const nextPageToFetch = observationList.length > 0 ? Math.ceil( observationList.length / perPage ) : 1;
   const [page, setPage] = useState( nextPageToFetch );
   const [userLogin, setUserLogin] = useState( null );
+  const [obsToUpload, setObsToUpload] = useState( [] );
 
-  const syncObservations = ( username ) => {
+  const syncObservations = ( username = null ) => {
```

Comment:
Why does this take a `username` at all? We can't sync obs if there isn't an authenticated user.

Reply:
This was my funky way of making this work when a user logs in for the first time. Normally, we would fetch the user login from the `getUsername` function, but that didn't seem to be working the first time a user logs in -- I assumed there was a timing issue where the username didn't have time to be stored and retrieved from RNSInfo by the time a user lands on the ObsList. So this is a special case where I'm passing the username via navigation parameters from the Login screen.

I'm guessing there's a cleaner way to do this.